pkg/clients/device/device.go

```diff
@@ -34,9 +34,9 @@ func NewClient(ctx context.Context, credentials []byte) (packngo.DeviceService,
 // CreateFromDevice ... TODO
 func CreateFromDevice(d *v1alpha1.Device) *packngo.DeviceCreateRequest {
 	return &packngo.DeviceCreateRequest{
-		HostName:     d.Spec.Hostname,
+		Hostname:     d.Spec.Hostname,
 		Plan:         d.Spec.Plan,
-		Facility:     d.Spec.Facility,
+		Facility:     []string{d.Spec.Facility},
```

Comment:
Should we change this to `[]string` in spec?

Reply:
We could do that, but I'm not sure how practical it will be.

Reply:
Can you specify multiple facilities in a create request? Does that provision multiple devices?

Reply:
You can specify multiple facilities but that only creates a single device in one of the provided facilities. For example { "facility": ["f3", "f2", "any"] } will try to assign to the facility f3, if there are no available f2, and so on. If "any" is not specified for "facility", the request will fail unless it can assign in the selected locations.

Reply:
Gotcha! I am fine with this behavior for now, but I would recommend that we reflect the exact possible configuration in the future, as the current implementation does expose the full functionality of the Packet API.